### PR TITLE
Strat edits

### PR DESCRIPTION
**Pull request by plehegar**
This clarifies the motivation/background section:

1. it's not replacing the current work of the W3C Strategy team, but contribute to it.
2. it's also useful for CGs as well.

As a side, it also adds useful links in the scope section.

cc @dennis-dingwei for review

This is part of https://github.com/w3c/strategy/issues/451

### CODE REVIEW

**Comment by plehegar** [comment]
cc @ianbjacobs  for the CG bits.

File: 2024/ig-exploration.html [hunk 1]
```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
...
+          The goal of Exploration Interest Group provides a platform to help the W3C community
           investigating emerging technology trends, analyzing their impacts on the evolution
           of Web technologies, and proposing ways for W3C to collaborate shaping the trends
           for the benefit of the Web users. The ways include organizing Workshops, publishing
           Reports and creating topic oriented Task Forces etc., assisting the Web community
           seeking innovative responses to continually strengthen the innovation of W3C and the
           creativity of Web technologies.
         </p>
+        <p>
+          The Exploration Interest Group also provides a forum for:
```
Comment (ianbjacobs) [inline 2024/ig-exploration.html]:
```suggestion
          Examples of who people may take advantage of this forum include:
```

File: 2024/ig-exploration.html [hunk 2]
```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
...
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
         <ul>
 <li>Monitor industry and technology trends that might impact the web;</li>
-<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, Workshops, Liaison Reports), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
-<li>Prioritize the trends and propose organizing Workshops, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
-<li>Propose chartering potential topics to the Strategy Funnel at the stage of incubation;</li>
+<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
```
Comment (ianbjacobs) [inline 2024/ig-exploration.html:175]:
```suggestion
<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. When asked by the W3C Team, work with them to invite W3C Liaisons to SDOs or other insightful SDO experts, reporting back on potential areas of investigations;</li>
```

File: 2024/ig-exploration.html [hunk 3]
```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
...
+<li>Prioritize the trends and propose <a href='https://github.com/w3c/strategy/labels/workshop'>organizing Workshops</a>, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
```
Comment (ianbjacobs) [inline 2024/ig-exploration.html]:
```suggestion
<li>Prioritize the trends and propose to the W3C Team <a href='https://github.com/w3c/strategy/labels/workshop'>organizing Workshops</a>, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
```

**Review by ianbjacobs — COMMENTED** [review]
Suggested some changes. Thanks!

File: 2024/ig-exploration.html (hunk 2 above)
Comment (plehegar) [reply 2024/ig-exploration.html:175]: Not sure about "When asked by the W3C Team". The IG can certainly propose.

Comment (ianbjacobs) [reply 2024/ig-exploration.html:175]: It's true that anyone can propose ideas to the Team. Do we need charter language in that case? How about deleting the second sentence? That makes it clearer that the scope of activity of the IG is to monitor the ecosystem, which seems like the true focus here.

File: 2024/ig-exploration.html [hunk 4]
```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
...
-<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, Workshops, Liaison Reports), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
-<li>Prioritize the trends and propose organizing Workshops, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
-<li>Propose chartering potential topics to the Strategy Funnel at the stage of incubation;</li>
+<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
+<li>Prioritize the trends and propose to the W3C Team <a href='https://github.com/w3c/strategy/labels/workshop'>organizing Workshops</a>, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
```
Comment (fantasai) [inline 2024/ig-exploration.html:176]: I'm not sure of adding "to the W3C Team" here. I think that's implied for Workshops, so not necessary to mention; but publishing Reports or creating Task Forces of this IG doesn't seem to need to be proposed "to the Team" -- the IG can just do it.

Comment (ianbjacobs) [reply 2024/ig-exploration.html:176]: I think there's no need to mention creation of task forces in the charter. The IG can organize its work however it wants.

**Comment by plehegar** [comment]
I'm closing this pull request.

There is new wording in https://github.com/w3c/strategy/issues/451#issuecomment-2520551409 which is the result of conversations with Ian and Dingwei.

**Pull request closed without merging**